apps/frontend/src/app/Solver/GOSolver/solveLP_simplex.ts

```diff
@@ -0,0 +1,119 @@
+/**
+ * Solve a Linear Program defined by:
+ *              min  c^T x
+ *               x
+ *   Subject to:     Ax <= b
+ *                    x >= 0
+ *
+ * Implemented according to the Simplex Method (Sec 4) of:
+ *   Ferguson, https://www.math.ucla.edu/~tom/LP.pdf
+ *
+ * Does not implement any cycle detection, though that *shouldnt* a problem for GO's use
+ *   case. This algorithm may be fairly numerically unstable though, use with care & always
+ *   try to verify the solution.
+ *
+ * @param c        Objective vector
+ * @param Ab       Constraints matrix with thresholds. Inputted in block form [A, b]
+ * @returns        the optimal solution x
+ */
+export function solveLP(c: number[], Ab: number[][]) {
+  const rows = Ab.length + 1
+  const cols = Ab[0].length
+
+  const tableau = Array(rows).fill(0).map(_ => Array(cols).fill(0))
+  Ab.forEach((Ai, i) => Ai.forEach((Aij, j) => tableau[i][j] = Aij))
+  c.forEach((cj, j) => tableau[rows - 1][j] = cj)
+
+  const ijTrack: { i: number, j: number }[] = []
```

Comment:
Is there a more descriptive name for this thing?

Reply:
renamed to pivotRecord

Reply:
Is this the term used in the referenced link? if not, please provide a brief description.